### PR TITLE
Update single-to-multi-master.md

### PR DESCRIPTION
**Pull request by jeyglk**
* Make the distinction between `<availability-zone>` and `<availability-zone-name>`, since they can be different (and actually, they are, by default.) it can lead to confusion.
* Add examples
* Fix docker stop command

### CODE REVIEW

File: docs/single-to-multi-master.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ Please follow all the backup steps before attempting it. Please read the
 [etcd admin guide](https://github.com/coreos/etcd/blob/v2.2.1/Documentation/admin_guide.md)
 before attempting it.
 
+We can migrate from a single-master cluster to a multi-master cluster, but this is a complicated operation. It is easier to create a multi-master cruster using Kops (described [here](https://github.com/kubernetes/kops/blob/master/docs/high_availability.md)). If possible, try to plan this at time of cluster creation.
```
Comment (gzzo) [inline docs/single-to-multi-master.md]: You have a typo here "cruster" should be "cluster"

Comment (jeyglk) [reply docs/single-to-multi-master.md]: oops, thanks, corrected

**Comment by mikesplain** [comment]
Good eye @gzzo 
/lgtm

File: docs/single-to-multi-master.md [hunk 2]
```diff
@@ -91,25 +104,55 @@ reference a third one, even if we have not created it yet.*
...
 ## 3 - Add a new master
 
 ### a - Add a new member to the etcd clusters
 
 **The clusters will stop to work until the new member is started**.
 
 ```bash
-$ kubectl --namespace=kube-system exec etcd-server-ip-172-20-36-161.ec2.internal -- etcdctl member add etcd-<availability-zone2> http://etcd-<availability-zone2>.internal.example.com:2380
-$ kubectl --namespace=kube-system exec etcd-server-events-ip-172-20-36-161.ec2.internal -- etcdctl --endpoint http://127.0.0.1:4002 member add etcd-events-<availability-zone2> http://etcd-events-<availability-zone2>.internal.example.com:2381
+$ kubectl --namespace=kube-system exec etcd-server-ip-172-20-36-161.ec2.internal -- etcdctl member add etcd-<availability-zone2-name> http://etcd-<availability-zone2-name>.internal.example.com:2380 \
```
Comment (chrislovecnm) [inline docs/single-to-multi-master.md]: Nit picks that you can follow up in another or if you like. Looks grear btw

I hate, and I know I have done it before the ‘$’ sign with the prompt makes copy paste a pain.

Can we get the name of the pod via a label? That way we do not have to have a name the is unique in the examples.

**Pull request merged**